### PR TITLE
fixed Confusing Included Concepts count

### PR DESCRIPTION
**Pull request by johnSamilin**
fixes https://github.com/OHDSI/WebAPI/issues/737

### CODE REVIEW

File: js/components/conceptsetInclusionCount/conceptsetInclusionCount.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ define([
 		getInclusionCount() {
 			this.countLoading(true);
 			conceptSetApi.getInclusionCount(this.conceptSetExpression())
-				.then(({data}) => this.inclusionCount(data))
+				.then(({data}) => this.inclusionCount(Number.isInteger(data) ? data : 0))
```
[the remark this answers is not in the document]
Comment (pavgra) [reply js/components/conceptsetInclusionCount/conceptsetInclusionCount.js]: Let's show an explicit message as @mpozhidaeva suggests (e.g. "Cannot load Included concepts due to problems with Vocabulary daimon")

**Comment by pavgra** [comment]
@johnSamilin , changed the vocabulary schema to incorrect one as described in the issue and cannot even reach the page:

image

**Comment by johnSamilin** [comment]
@pavgra could you check again?

**Comment by pavgra** [comment]
@johnSamilin , from the Concept Sets section point-of-view looks good. However, we also have concept sets embedded into Cohort Definitions. Would be great to handle such case there also

image

File: js/pages/concept-sets/components/tabs/included-conceptsets.html [hunk 2]
```diff
@@ -16,7 +16,10 @@
 <!-- ko if: $root.loadingIncluded -->
   <loading></loading>
 <!-- /ko -->
-<!-- ko ifnot: $root.loadingIncluded -->
+<!-- ko if: $component.hasErrors -->
+  <empty-state params="message: 'Could not load included concept sets'"></empty-state>
```
Comment (pavgra) [inline js/pages/concept-sets/components/tabs/included-conceptsets.html]: With the `alert` approach, is there a need in the change? In which case are we going to reach the view?

File: js/Model.js [hunk 3]
```diff
@@ -399,20 +399,18 @@ define(
...
+							throw new Error();
+						}
+						this.conceptSetInclusionIdentifiers(info);
+						this.currentIncludedConceptIdentifierList(info.join(','));
+						this.conceptSetInclusionCount(info.length);
+					};
 				this.resolvingConceptSetExpression(true);
-				const resolvingPromise = httpService.doPost(sharedState.vocabularyUrl() + 'resolveConceptSetExpression', expression);
-				resolvingPromise.then(callback);
-				resolvingPromise.catch((err) => {
```
Comment (pavgra) [inline js/Model.js]: What about direct calls of the method? e.g. from `Application.js`. Why don't you leave the `catch`?

**Pull request merged**